rbc/irtools.py

```diff
@@ -362,6 +362,27 @@ def compile_IR(ir):
     return engine
 
 
+def _fflush(builder):
+    int8_t = ir.IntType(8)
+    fflush_fnty = ir.FunctionType(int32_t, [int8_t.as_pointer()])
+    fflush_fn = builder.module.get_or_insert_function(
+        fflush_fnty, name='fflush')
+
+    builder.call(fflush_fn, [int8_t.as_pointer()(None)])
+
+
+@extending.intrinsic
+def fflush(typingctx):
+    """fflush that can be called from Numba jit-decorated functions.
+    """
+    sig = nb_types.void(nb_types.void)
+
+    def codegen(context, builder, signature, args):
+        _fflush(builder)
```

Comment:
Could you inline `_fflush` as it is not used elsewhere?

Reply:
It is not possible to call `flush` when generating code. Inlining `_fflush` makes it available to only jitted code. But having it, allows one to call it from a `codegen()` function.

Reply:
Sorry for the confusion. By inlining here I mean to insert the `_fflush` Python code to `codegen` so that the function `_fflush` will be eliminated.